File: matiasmasso/Mat.NET/DT2/DTOContactDoc.vb
Public Class DTOContactDoc
    Inherits DTOBaseGuid
    Property Contact As DTOContact
    Property Type As Types
    Property Fch As Date
    Property Ref As String
    Property DocFile As DTODocFile
    Property Obsoleto As Boolean

    Public Enum Types
        NotSet
        NIF
        Escriptura
        Contracte
        SeguretatSocial
        Academic
        Tarjeta
        Retencions
        Model_145
    End Enum

    Public Sub New(ByVal oGuid As System.Guid)
        MyBase.New(oGuid)
    End Sub

    Public Sub New()
        MyBase.New()
    End Sub

End Class

File: matiasmasso/Mat.NET/DTO/DTOContactDoc.cs
using System;

namespace DTO
{
    public class DTOContactDoc : DTOBaseGuid
    {
        public DTOContact Contact { get; set; }
        public Types Type { get; set; }
        public DateTime Fch { get; set; }
        public string Ref { get; set; }
        public DTODocFile DocFile { get; set; }
        public bool Obsoleto { get; set; }

        public enum Types
        {
            NotSet,
            NIF,
            Escriptura,
            Contracte,
            SeguretatSocial,
            Academic,
            Tarjeta,
            Retencions,
            Model_145
        }

        public DTOContactDoc(System.Guid oGuid) : base(oGuid)
        {
        }

        public DTOContactDoc() : base()
        {
        }
    }
}
